### ISSUE TITLE
Order rejected - not enough tradable balance

### ISSUE THREAD

**Issue by me21** [wavesplatform/Waves#910]
My Waves account is 3PMs8yKFkFgcFGV8PcMEW7dbevomDtdnQuK. I want to sell 25.57 WTRY which I have in the wallet. When doing so with PyWaves, I get the following error:

```
[ERROR] Order Rejected - Not enough tradable balance: Map(2mX5DzVKWrAJw8iwdJnV2qtoeVG9h5nTDpTqC1wb1WEN -> 2556, WAVES -
> 63309902), needs: OpenPortfolio(Map(WAVES -> 300000, 2mX5DzVKWrAJw8iwdJnV2qtoeVG9h5nTDpTqC1wb1WEN -> 2557))
```

Why tradable balance is 25.56 whereas there's 25.57 WTRY in the wallet? There is no outstanding orders for my account.

The same error is visible in the Web client (beta). Not enough tradable balance.

**Comment by alexeykiselev** [comment]
Thank you for the report!

**Comment by me21** [comment]
Right now WBTC tradable balance is reported as 0.00076616, total balance is 0.00076617. WTRY tradable balance is 0, total balance is 0.01.
Can I check if I have any open orders for these assets? I think I haven't, but it's always good to make sure.

**Comment by monroid** [comment]
@me21 Hi there, 
we suppose your issue will be close by https://github.com/wavesplatform/Waves/pull/1295. So stay tuned, we'll inform you what version has the fix.

**Comment by Saske-SXAS** [comment]
Has this issue been resolved?

**Comment by phearnot** [comment]
This has been fixed in 0.14. Feel free to reopen this issue if you encounter similar problems.
